UXO_Kismet_SF_3.tex

```diff
@@ -0,0 +1,363 @@
+        \end{checklist}
+    \end{checklist}
+    \item \heat{}: Open the Tank Heating Valve.
+    \item \daq{}: Proceed only when the oxidizer tank pressure is at least 800 psi.
+    \begin{checklist}[label=$\bullet$]
+        \item If the oxidizer tank pressure does not reach 800 psi:
+        \begin{checklist}
+            \item \heat{}: Close the Tank Heating Valve.
+            \item \primary{}: Open the Line Vent Valve using the ropes.
+            \item \ops{}: Proceed only when the oxidizer tank has fully vented.
+            \item \primary{} and \secondary{}: Approach the test plumbing.
+            \item \primary{}: Close the Cylinder Valve.
+            \item \control{}: Open the Tank Vent Valve.
+            \item \control{}: Open the Remote Fill Valve.
+            \item \daq{}: Confirm that the Oxidizer Tank Pressure Transducer is reading atmospheric pressure.
+            \item \ops{}: Abort test procedures and revisit water jacket setup.
+        \end{checklist}
+    \end{checklist}
+    \item \heat{}: Close the Tank Heating Valve.
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \control{}: Perform engine startup procedure:
+    \begin{checklist}
+        \item Arm the Primary Ignition switch.
+        \item Hold down the Fire button until black smoke is observed.
+        \begin{checklist}[label=$\bullet$]
+            \item In the event of a failed ignition (smoke not observed within 1 minute):
+            \begin{checklist}
+                \item \control{}: Disarm the Primary Ignition switch.
+                \item \control{}: Arm the Secondary Ignition switch.
+                \item \ops{}: Revisit ignition procedure.
+            \end{checklist}
+            \item In the event of a second failed ignition (smoke not observed within 1 minute):
+            \begin{checklist}
+                \item \control{}: Disarm the Secondary Ignition switch.
+                \item \primary{}: Open the Line Vent Valve using the ropes.
+                \item \ops{}: Proceed only when the oxidizer tank has fully vented.
+                \item \primary{} and \secondary{}: Approach the test plumbing.
+                \item \primary{}: Close the Cylinder Valve.
+                \item \control{}: Open the Remote Fill Valve.
+                \item \control{}: Open the Tank Vent Valve.
+                \item \daq{}: Confirm that the Oxidizer Tank Pressure Transducer is reading atmospheric pressure.
+                \item \ops{}: Abort test procedures and proceed to teardown.
+            \end{checklist}
+        \end{checklist}
+        \item \control{}: Start the engine by opening the Injector Valve.
+    \end{checklist}
+    \item \textbf{ALL}: Observe the plume:
+    \begin{checklist}[label=$\bullet$]
+        \item If any unexpected events occur during the engine firing:
+        \begin{checklist}
+            \item \primary{}: Open the Line Vent Valve using the ropes.
```

Comment:
Can we hear each other while the plume is going? Not sure how this will be signalled?

I don't think having this option really buys us much, what's the circumstance under which we'd want to open the vent valve?

Reply:
That's fair. This section has mostly just been left in for legacy reasons, but realistically I can't imagine us ever observing something unexpected, making the decision to shut off the engine, and successfully coordinating it in the span of the burn time. Removed.